### DISCUSSION TITLE
Bring back hold-to-confirm to pause menu

### DISCUSSION THREAD

**Discussion by Bruno5430** [ppy/osu#30664] [Ideas] [answered]
I play on a small keyboard and sometimes i hit `Esc` on accident, and the old delay stopped me from doing that
This happens to me even on stable, and I don't want to remove the key entirely when I'm playing

[bring-back-pause-delay.webm](https://github.com/user-attachments/assets/d6e7a476-ad0a-45e9-bd87-4d3f4f914471)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/30878
